src/styles/blocks/gallery.scss

```
@@ -0,0 +1,57 @@
+.gallery {
+  margin-bottom: -40px;
+
+  &__line {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
```

```suggestion
    grid-template-columns: 1fr 1fr;
```
You can give here 1fr 1fr by default and change it with modifiers (as you've already done).
Then you can get rid of class `gallery__line--2` because it also have 1fr 1fr